test/test_helper.rb

```diff
@@ -42,10 +42,23 @@ def create_image_blob(filename: "racecar.jpg", content_type: "image/jpeg")
         filename: filename, content_type: content_type
     end
 
-    def assert_same_image(fixture_filename, variant)
-      assert_equal \
-        File.binread(File.expand_path("../fixtures/files/#{fixture_filename}", __FILE__)),
-        File.binread(variant.service.send(:path_for, variant.key))
+    def assert_same_image_dimensions(fixture_filename, variant)
+      fixture_and_variant fixture_filename, variant do |expected_image, actual_image|
+        assert_equal expected_image.width, actual_image.width
+        assert_equal expected_image.height, actual_image.height
+      end
+    end
+
+    def assert_same_image_colorspace(fixture_filename, variant)
+      fixture_and_variant fixture_filename, variant do |expected_image, actual_image|
+        assert_equal expected_image.colorspace, actual_image.colorspace
+      end
+    end
+
+    def fixture_and_variant(fixture_filename, variant)
+      expected_image = MiniMagick::Image.open(File.expand_path("../fixtures/files/#{fixture_filename}", __FILE__))
+      actual_image   = MiniMagick::Image.open(variant.service.send(:path_for, variant.key))
+      yield expected_image, actual_image
```

Comment:
Could be split out into `read_image_fixture` and `read_image_variant`.

Would also be nice if the fixture reader could use Rails' `file_fixture`:

```ruby
def read_image_fixture(filename)
  MiniMagick::Image.open(file_fixture(filename).to_s)
end
```